composer.json

```
@@ -16,8 +16,8 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
-        "illuminate/support": "5.8.* || ^6.0",
+        "php": "^7.1.3 | ^8.0",
+        "illuminate/support": "5.8.* || ^6.0" || '^7.0' || '8.0',
```

Wrong syntax here, everything should be between the double quotes (`"`), and you won't need the single quotes anymore if you do that:

```suggestion
        "illuminate/support": "5.8.* || ^6.0 || ^7.0 || ^8.0",
```